archivebox/config.py

```diff
@@ -144,12 +144,19 @@
                                                                 '--no-call-home',
                                                                 '--write-sub',
                                                                 '--all-subs',
-                                                                '--write-auto-sub',
+                                                                # There are too many of these and youtube
+                                                                # throttles you with HTTP error 429
+                                                                #'--write-auto-subs',
                                                                 '--convert-subs=srt',
                                                                 '--yes-playlist',
                                                                 '--continue',
-                                                                '--ignore-errors',
+                                                                # This flag doesn't exist in youtube-dl
+                                                                # only in yt-dlp
                                                                 '--no-abort-on-error',
+                                                                # --ignore-errors must come AFTER
+                                                                # --no-abort-on-error
+                                                                # https://github.com/yt-dlp/yt-dlp/issues/4914
```

Comment:
The order of these flags is---confusingly---quite important.

https://github.com/yt-dlp/yt-dlp/issues/4914